modules/@apostrophecms/doc-type/ui/apos/components/AposDocLocalePicker.vue

```diff
@@ -70,19 +77,47 @@ async function open() {
   if (!props.docId) {
     return;
   }
-  const docs = await apos.http.get(
+  try {
+    const docs = await apos.http.get(
     `${props.moduleOptions.action}/${props.docId}/locales`, { busy: true }
-  );
-  localized.value = Object.fromEntries(
-    docs.results
-      .filter(doc => doc.aposLocale.endsWith(':draft'))
-      .map(doc => [ doc.aposLocale.split(':')[0], doc ])
-  );
+    );
+    localized.value = Object.fromEntries(
+      docs.results
+        .filter(doc => doc.aposLocale.endsWith(':draft'))
+        .map(doc => [ doc.aposLocale.split(':')[0], doc ])
+    );
+
+    await checkCreatePermission();
+  } catch (err) {
+    console.err(err);
```

Comment:
we don't want to re-throw?

Reply:
Right I don't need two try catch since there is one above.